src/util/psbt/error.rs

```diff
@@ -85,6 +87,7 @@ impl fmt::Display for Error {
             Error::InvalidKey(ref rkey) => write!(f, "invalid key: {}", rkey),
             Error::InvalidProprietaryKey => write!(f, "non-proprietary key type found when proprietary key was expected"),
             Error::DuplicateKey(ref rkey) => write!(f, "duplicate key: {}", rkey),
+            Error::FutureVersion(ref rkey) => write!(f, "unrecognized PSBT version: {}", rkey),
```

Comment:
Needs to be `write_err!`

Reply:
Should all of these match arms output `write_err!` as well? In a separate commit, of course.

Reply:
No, `raw::Key` is not an error type. Also calling this one `rkey` is misleading for the same reason, it should be `error`.

Reply:
No, `raw::Key` is not an error type. Also calling this one `rkey` is misleading for the same reason, it should be `error`.